resources/lemmas.md

```diff
@@ -333,6 +333,29 @@ The other rules are similar.
     rule chop(I) => I requires 0 <=Int I andBool I <Int pow256
 ```
 
+### `-Word` Reduction
+
+```k
+    rule chop ( W0:Int +Int W1:Int ) -Word W1:Int => chop ( W0 )
+    requires #rangeUInt(256, W0) andBool #rangeUInt(256, W1)
+```
+
+### `modInt` Reduction
+
+```k
+    rule X:Int modInt 1461501637330902918203684832716283019655932542976 => X
```

Comment:
There is a more general lemma for modInt in gnosis/verification.k and other versions of verification.k. Please use that version.

Reply:
Done